consensus/propagation/catchup.go

```diff
@@ -1,45 +1,54 @@
 package propagation
 
 import (
+	proptypes "github.com/tendermint/tendermint/consensus/propagation/types"
 	"github.com/tendermint/tendermint/p2p"
-	"github.com/tendermint/tendermint/types"
 )
 
-// sendPsh sends the partset header to the provided peer.
-// This method will:
-// - get the proposal with the provided height and round
-// - get the patset header from the block meta?
-// - send the partset header to the provided peer
-// TODO rename this Psh to something less Psh
+// sendPsh sends the part set header to the provided peer.
+// TODO check if we still need this?
```

Comment:
since we won't support gaps, this is not needed @evan-forbes ?

Reply:
also, what do you think of renaming the proposal cache to commitment cache or similar?

Reply:
sure we can rename it

yeah I think we don't need this for now

Reply:
50ebeb29ce9af0165ba5fc474fd04d1c700acbbb

Reply:
46de301477ae277a0852d347e32780578721ca8c